test/unit/operations/get_more.test.ts

```diff
@@ -127,11 +93,110 @@ describe('GetMoreOperation', function () {
         operation.execute(server2, session, callback);
       });
     });
+
+    context('command construction', () => {
+      const cursorId = Long.fromBigInt(0xffff_ffffn);
+      const namespace = ns('db.collection');
+      const server = new Server(new Topology([], {} as any), new ServerDescription(''), {} as any);
+
+      it('should build basic getMore command with cursorId and collection', async () => {
+        const getMoreOperation = new GetMoreOperation(namespace, cursorId, server, {});
+        const stub = sinon.stub(server, 'command').callsFake((_, __, ___, cb) => cb());
+        await promisify(getMoreOperation.execute.bind(getMoreOperation))(server, undefined);
+        expect(stub).to.have.been.calledOnce;
```

Comment:
I believe this can be `to.have.been.calledOnceWith(sinon.match.any, { ...expected })`
it's just a bit cleaner than the manual extraction of the arguments;

for the partial assertions below, you can use `sinon.match.has(property[, expectation])`